src/app/screens/routines/exercises-revision/components/exercise-item/index.tsx

```diff
@@ -0,0 +1,99 @@
+import { Plus, Trash } from 'lucide-react-native';
+import React, { FC } from 'react';
+import { useFieldArray } from 'react-hook-form';
+import { View } from 'react-native';
+
+import {
+  Button,
+  ControlledSelect,
+  ControlledTextInput,
+  IconButton,
+} from 'src/app/components';
+import { AccordionItem } from 'src/app/components/accordion';
+
+import { ExerciseContainer, InputContainer, Row } from './styles';
+
+interface Props {
```

Comment:
Move all the types into a file and named the interfaces and types with the correct name, not just Props